app/adapters/application.js

```diff
@@ -0,0 +1,29 @@
+import DS from 'ember-data';
+// require('electron');
+import database from './database';
```

Comment:
We tried it both ways but couldn't tell since we just kept getting the same error.